packages/gatsby/src/utils/webpack.config.js

```diff
@@ -232,7 +233,9 @@ module.exports = async (
 
       plugins.virtualModules(),
       new BabelConfigItemsCacheInvalidatorPlugin(),
-    ]
+      process.env.GATSBY_WEBPACK_LOGGING?.split(`,`)?.includes(stage) &&
```

Comment:
NIT: other places skip the `.split()` part, so this is just making this check consistent across the code
```suggestion
      process.env.GATSBY_WEBPACK_LOGGING?.includes(stage) &&
```

Reply:
I need to split because we have `develop` and `develop-html`